src/main/java/sx/blah/discord/api/internal/json/event/ReactionEventResponse.java

```diff
@@ -0,0 +1,12 @@
+package sx.blah.discord.api.internal.json.event;
+
+import sx.blah.discord.api.internal.json.objects.ReactionEmojiObject;
+
+public class ReactionEventResponse {
+
+	public String user_id;
```

Comment:
Not entirely necessary but it would be good to have documentation. (This goes for other added json objects as well)